geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/AcceptorImpl.java

```diff
@@ -1498,6 +1493,17 @@ protected void handleNewClientConnection(final Socket socket,
     }
   }
 
+  private boolean initializeClientPools(Socket socket, CommunicationMode communicationMode) {
```

Comment:
This method name doesn't seem to quite capture the functionality, maybe something like handOffClientQueueInitialization?